docker-compose.yml

```diff
@@ -67,44 +67,46 @@ services:
       - --batch_size=50
       - --force_master=true
       - --alsologtostderr
+    ports:
+      - "8091"
```

Comment:
No longer need port 8092?

Reply:
All the Trillian processes expose 8090 and 8091.  To avoid port clashing I was mapping this to 8092 so an external initialization script could create a Trillian Log. Now that I'm initializing things with a docker init container, these ports no longer need to be exposed. 

In this configuration Kubernets will make the port available internally to the cluster, and docker-compose will pick a random host port to map 8091 to.